src/Components/Query_Loop_Controller.php

```diff
@@ -22,7 +23,9 @@ abstract protected function prepare_posts_for_display( array $posts = [], bool $
 	public function __construct( $block ) {
 		$this->block      = (array) $block;
 		$this->query_loop = (array) get_field( Query_Loop::QUERY_LOOP );
-		$this->cta        = (array) get_field( Featured_News_Block::CTA_FIELD );
+		$this->cta        = (array) get_field( Featured_News_Block::CTA_FIELD )
```

Comment:
This should be moved from there to child class. E.g `Media_Coverage_Controller` will call 
```
public function __construct($block) {
        $this->cta = (array) get_field( Media_Coverage_Block::CTA_FIELD ) ?: [];
        
        parent::__construct($block);
    }
```
Same for Featured News. I missed on the last review but will have more blocks with that and will be completely unreadable

Reply:
Ah, thank you! Updated, please take another look.